src/lib/components/wizard/st-import-steps/StepImportReview.svelte

```
@@ -65,13 +83,135 @@
+        id: 'classification',
+        label: 'Classifying chapters',
+        status: done ? 'completed' : (active ? 'active' : 'pending'),
+        details: active
+          ? `Classifying: chapter ${chapterizationClassificationProgress.current}/${chapterizationClassificationProgress.total}`
+          : (done ? 'Done' : 'Waiting in queue...')
+      })
+    }
+
+    // 5. Lorebook update
+    if (chapterizeIncludeLorebook) {
+      const active = chapterizationStatus !== null &&
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)